code/pagetypes/Forum.php

```diff
@@ -731,9 +731,13 @@ function BBTags() {
 	 *
 	 * @return Form Returns the post message form
 	 */
-	function PostMessageForm($addMode = false, $post = false) {
-		$thread = false;
+	function PostMessageForm($addMode = true, $post = false) {
```

Comment:
Will look to cherry pick and revert this. If we purely just fix the bug perhaps that warrants this being a patch release of 0.7 ?